app/javascript/hooks/mutations/rooms/StartSession.jsx

```diff
@@ -0,0 +1,25 @@
+
+import { useMutation } from "react-query";
+import axios, { ENDPOINTS } from "../../../helpers/Axios";
+
+export default function usePostStartSession(room_id) {
```

Comment:
I would call this something like `useCreateMeeting`

Reply:
So the naming pattern would be `use[CRUD][Subject]` rather than `use[Method][Subject]`?

Reply:
Yeah